uqcsbot/minecraft.py

```diff
@@ -119,7 +167,6 @@ async def mcadmin(self, interaction: discord.Interaction, command: str):
             for split in split_response[1:]:
                 await interaction.followup.send(f"```{split}```")
 
-        # Just to be safe, send this to the admin log channel as well.
```

Comment:
If you could re-add this comment (I assume removing it was just a mistake) that'd be nice.

Reply:
Added this back in, thanks for catching that